<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changePassword</class>
 <widget class="QWidget" name="changePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="beforePWD">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>原密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstPWD">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入新密码,长度3-15</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="secondPWD">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请确认新密码,长度3-15</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sureButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>确认修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="PWDtitle">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
